copilot/fsd-authenticator/manifest.yml

```diff
@@ -95,18 +95,19 @@ environments:
         value: 80
       requests: 30
       response_time: 2s
-  production:
+  prod:
     http:
-      alias: authenticator.production.access-funding.levellingup.gov.uk
-      # alias: ["authenticator.production.access-funding.levellingup.gov.uk", "authenticator.access-funding.levellingup.gov.uk"] # For go live
+      alias: authenticator.prod.access-funding.levellingup.gov.uk
+      # alias: ["authenticator.prod.access-funding.levellingup.gov.uk", "authenticator.access-funding.levellingup.gov.uk"] # For go live
     variables:
       ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK: false
       COOKIE_DOMAIN: ".levellingup.gov.uk"
-      AUTHENTICATOR_HOST: "https://authenticator.production.access-funding.levellingup.gov.uk"
-      APPLICANT_FRONTEND_HOST: "https://frontend.production.access-funding.levellingup.gov.uk"
-      ASSESSMENT_FRONTEND_HOST: "https://assessment.production.access-funding.levellingup.gov.uk"
+      AUTHENTICATOR_HOST: "https://authenticator.prod.access-funding.levellingup.gov.uk"
```

Comment:
_In theory_ these could be their 'live' counterparts, similar to POST_AWARD_FRONTEND_HOST and POST_AWARD_SUBMIT_HOST - but they'd both lead to the same place anyway. So...this is mainly just a note for posterity